README.md

```diff
@@ -71,6 +72,15 @@ wrapper classes or even your ENC (if it supports param classes). For example:
       server_external_nodes => '',
     }
 
+    # Want to integrate with an existing PuppetDB?
+    class { '::puppet':
+      server               => true,
+      server_puppetdb_host => 'mypuppetdb.example.com',
+      server_reports       => 'puppetdb',
```

Comment:
As foreman got optional, I presume not, but I wouldn't mind having it also in just as example.